website/docs/r/policy_predefined_security_policy.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+subcategory: "Policy - Firewall"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_predefined_security_policy"
+description: A resource to update Predefined (Default) Security Security Policies.
+---
+
+# nsxt_policy_predefined_security_policy
+
+This resource provides a method to modify default Security Policy and its rules.
+This can be default layer2 policy or default layer2 policy. Maximum one resource
```

Comment:
default layer2 policy or default layer3 policy

Reply:
Thanks for the catch! Done